server/android/src/lib.rs

```diff
@@ -1 +1,2 @@
+#![allow(clippy::all)]
```

Comment:
Why did you disable them for Android?

Reply:
Sorry, I just disabled it for now... will enable again when I do the rest of the clippy fixes 👀

Reply:
ok cool, could you please take it out of the PR though so that we don't forget?